scripts/migrate_wwcode_test_db.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Can you add a line to create database if it doesn't exist? Getting an error for that now.